app/autoscaler.py

```diff
@@ -16,7 +16,7 @@
 
 
 def _get_redis_url():
-    if "VCAP_SERVICES" in os.environ:
+    if "REDIS_URL" not in os.environ:
         try:
             return json.loads(os.environ["VCAP_SERVICES"])["redis"][0]["credentials"]["uri"]
```

Comment:
The logic in this file is a bit different.

The `_get_redis_url` function returns a value.

Before your PR it says:
If VCAP services is set, then return the value pulled out of the PaaS VCAP environment variable
Else return `redis://redis.local`

The behaviour we want is
If REDIS_URL is set as an environment variable, then return it
Else if VCAP services is set, then return the value pulled out of the PaaS VCAP environment variable
Else return `redis://redis.local`

That will allow the REDIS_URL we provide to override. 

It will need different logic to what you are proposing. Have a look at the logic again please and see if you can adapt to meet the suggested behaviour above. And if you get stuck, give a shout out on slack and someone should be able to pair with you on this!

Reply:
Please have a look at the logic now.